main/bubblewrap/template.py

```diff
@@ -1,18 +1,14 @@
 pkgname = "bubblewrap"
-pkgver = "0.8.0"
+pkgver = "0.9.0"
 pkgrel = 0
-build_style = "gnu_configure"
-make_check_args = ["-j1"]
-hostmakedepends = ["pkgconf", "xsltproc", "docbook-xsl-nons"]
+build_style = "meson"
+hostmakedepends = ["meson", "pkgconf", "xsltproc", "docbook-xsl-nons"]
 makedepends = ["libcap-devel"]
 checkdepends = ["bash", "libcap-progs"]
 pkgdesc = "Unprivileged sandboxing tool"
 maintainer = "q66 <q66@chimera-linux.org>"
 license = "LGPL-2.0-or-later"
 url = "https://github.com/containers/bubblewrap"
 source = f"{url}/releases/download/v{pkgver}/{pkgname}-{pkgver}.tar.xz"
-sha256 = "957ad1149db9033db88e988b12bcebe349a445e1efc8a9b59ad2939a113d333a"
-tool_flags = {"CFLAGS": ["-Wno-error,-Wformat-nonliteral"]}
```

Comment:
What's the point of this exactly?

Reply:
back in 0.5 there was a -Werror=format-nonliteral amongst other things so that one was turned off prolly cause it failed

Reply:
right that explains the -Wno-error, but what about -Wformat-nonliteral?

Reply:
that part looks like a typo